test/transaction_test.rb

```diff
@@ -3,17 +3,22 @@
 require_relative '../lib/transaction'
 
 class TransactionTest < MiniTest::Test
+  attr_reader :parent, :transaction
+  def setup
+    sales_engine = SalesEngine.new
+    @parent = TransactionRepository.new('test/support/sample_transactions.csv', sales_engine)
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.